objects/tag.c

```diff
@@ -139,7 +139,6 @@ tag_view(lua_State *L, int udx, bool view)
     {
         tag->selected = view;
         banning_need_update();
-        ewmh_update_net_current_desktop();
```

Comment:
Got it, thank you.

I'll move tag class to header then and push the next set of changes later today probably.